src/rational.jl

```diff
@@ -66,7 +66,7 @@ Base.:*(r::RationalPoly, p::APL)          = (r.num * p) / r.den
 Base.:*(α, r::RationalPoly)               = (α * r.num) / r.den
 Base.:*(r::RationalPoly, α)               = (r.num * α) / r.den
 
-Base.zero(::RationalPoly{NT}) where {NT} = zero(NT)
-Base.zero(::Type{RationalPoly{NT, DT}}) where {NT, DT} = zero(NT)
-Base.one(::RationalPoly{NT}) where {NT} = one(NT)
-Base.one(::Type{RationalPoly{NT, DT}}) where {NT, DT} = one(NT)
+Base.zero(::RationalPoly{NT}) where {NT} = zero(NT) / one(NT)
```

Comment:
In fact, this is incorrect, it should be `one(DT)` instead of `one(NT)`. The easiest might be to do `Base.zero(r::RationalPoly) = zero(typeof(r))`

Reply:
You're right, I was sloppy there. I'm going to change it.